test/smoke/testApps/MongoDB/src/main/resources/ApplicationInsights.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" schemaVersion="2014-05-30">
+
+    <!-- The key from the portal: -->
+
+    <InstrumentationKey>00000000-0000-0000-0000-0FEEDDADBEEF</InstrumentationKey>
+
+    <SDKLogger type="CONSOLE">
+        <enabled>true</enabled>
+        <UniquePrefix>JavaSDKLog</UniquePrefix>
+    </SDKLogger>
+
+    <QuickPulse enabled="false" />
+
+    <!-- HTTP request component (not required for bare API) -->
+
+    <TelemetryModules>
+        <Add type="com.microsoft.applicationinsights.web.extensibility.modules.WebRequestTrackingTelemetryModule" />
+        <Add type="com.microsoft.applicationinsights.web.extensibility.modules.WebSessionTrackingTelemetryModule" />
+        <Add type="com.microsoft.applicationinsights.web.extensibility.modules.WebUserTrackingTelemetryModule" />
+    </TelemetryModules>
+
+    <!-- Events correlation (not required for bare API) -->
+    <!-- These initializers add context data to each event -->
+
+    <TelemetryInitializers>
+        <Add type="com.microsoft.applicationinsights.web.extensibility.initializers.WebOperationIdTelemetryInitializer" />
+        <Add type="com.microsoft.applicationinsights.web.extensibility.initializers.WebOperationNameTelemetryInitializer" />
+        <Add type="com.microsoft.applicationinsights.web.extensibility.initializers.WebSessionTelemetryInitializer" />
+        <Add type="com.microsoft.applicationinsights.web.extensibility.initializers.WebUserTelemetryInitializer" />
+        <Add type="com.microsoft.applicationinsights.web.extensibility.initializers.WebUserAgentTelemetryInitializer" />
+        <Add type="com.microsoft.applicationinsights.extensibility.initializer.docker.DockerContextInitializer" />
+    </TelemetryInitializers>
+
+    <Channel>
+        <EndpointAddress>http://fakeingestion:60606/v2/track</EndpointAddress>
```

Comment:
We should start using connection string. See #1059 for samples

Reply:
Update pushed